ItalianTranslation_PlanetCoaster.xml

```diff
@@ -4150,11 +4150,11 @@
 		<entry key="Research_Revealed_FR_Zozo" translation="Zozo"/>
 		<entry key="Research_Revealed_GoKarts" translation="Speed"/>
 		<!--C2_Scenery-->
-		<entry key="BuildingPartDesc_FAD_CoffeeGenericCup" translation="Alimenti di plastica senza marchio."/>
-		<entry key="BuildingPartDesc_FAD_GenericDonut3d" translation="Alimenti di plastica senza marchio."/>
-		<entry key="BuildingPartDesc_FAD_GenericEnergy3D" translation="Alimenti di plastica senza marchio."/>
+		<entry key="BuildingPartDesc_FAD_CoffeeGenericCup" translation="Tazza di caffè di plastica."/>
```

Comment:
Bisognerebbe verificare in game se questi vanno bene, hai visto dove appaiono?

Reply:
In realtà di questa entry ne esistono due copie.. il gioco attualmente legge la versione dalla linea 4325 in poi.. "Statua Generica Tazzina di Caffè" questa versione non viene trovata.. strano..